app/src/main/res/drawable/ic_search_results_device_tablet.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="16dp"
+    android:width="16dp"
+    android:viewportHeight="20"
+    android:viewportWidth="20">
+    <path android:fillColor="?attr/awesomeBarIndicatorFoundationColor"
+        android:fillType="nonZero"
+        android:pathData="m4,1h12c2.2091,0 4,1.7909 4,4v10c0,2.2091 -1.7909,4 -4,4h-12c-2.2091,0 -4,-1.7909 -4,-4v-10c0,-2.2091 1.7909,-4 4,-4z"/>
```

Comment:
It would be nice to minify the paths using a tool like [avocado](https://github.com/alexjlockwood/avocado) 😄